functions/display-logic.php

```diff
@@ -11,7 +11,7 @@
  * @author Cadie Stockman
  * @return string HTML markup for the story
  **/
-function display_story_callout( $story, $css_class='', $show_category=false, $thumbnail_size='frontpage-story-thumbnail', $heading='h3' ) {
+function display_story_callout( $story, $css_class='', $show_category=false, $thumbnail_size='frontpage-story-thumbnail', $heading='p' ) {
```

Comment:
```suggestion
function display_story_callout( $story, $css_class='', $show_category=false, $thumbnail_size='frontpage-story-thumbnail', $heading='h3' ) {
```

Based on what Roger says, we can put this back to an `h3` and _leave_ the `h3` class on the heading, but then update the specific [call of this function](https://github.com/UCF/Pegasus-Theme/blob/87a8cb43b95997f3a24c7270440aedfbefbe5e31/versions/v6/functions.php#L18) that gets executed when building the "In This Issue" section. It could be updated to read `echo display_story_callout( $story, '', false, 'single-post-thumbnail-3x2', 'p' );`

Reply:
In front-page.php page, span tag was passed as 5th parameter in display_story_callout function. So now only in this issue now is receiving the span tag.